src/load.lisp

```diff
@@ -222,6 +246,24 @@
       (setq code-stor nil))
     (values)))
 
+;;; error message handle path
+(defun err_p3010_ (condition)
```

Comment:
What is the meaning of p3010?  should we find a more clear name?

Reply:
>What is the meaning of p3010? should we find a more clear name?

Done